config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -0,0 +1,30 @@
+periodics:
+  pjh/kubernetes:
```

Comment:
periodics are not key'd by org/repo :-)

the checkout part is a bit different, https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/test-infra/janitors.yaml#L38-L42 might be a good exampple